README.md

```diff
@@ -135,6 +135,20 @@ end
 Configuration
 -------------
 
+### Identifiers
```

Comment:
Hi @johnmosesman,

The example initializer below contains examples of the default identifiers, and explanations about their customization. Just as the default Devise initializer, that file is intended to be read, and I don't think it is a good thing to duplicate these examples.

A note explaining that examples are contained in the example initializer could however be useful to make clearer that it is a file worth to read.